tests/hopr.py

```diff
@@ -273,6 +274,14 @@ async def get_tickets_statistics(self):
         """
         _, response = self.__call_api(TicketsApi, "tickets_get_statistics")
         return response
+    
+    async def get_ticket_price(self):
+        """
+        Returns the ticket price in wei.
+        :return: price: int
+        """
+        _, response = self.__call_api(TicketsApi, "tickets_get_price")
```

Comment:
hoprd-sdk-python would need to be generated first.

Reply:
Correct, but sdk generation fails on my side, already mentioned in the other PR #5766:
```
File ./hoprd-sdk-python/ is not a regular file -- refusing to patch
18 out of 18 hunks ignored -- saving rejects to file ./hoprd-sdk-python/.rej
File ./hoprd-sdk-python/ is not a regular file -- refusing to patch
patch unexpectedly ends in middle of line
patch: **** malformed patch at line 1287:  

make: *** [Makefile:592: generate-python-sdk] Error 2
```

Reply:
Gonna fix that.